### DISCUSSION TITLE
Menambahkan tab menu di shiny

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#557] [Q&A] [answered]
Bagaimana cara menambahkan tab menu pada bagian atas shiny ?

Tampilanya mau seperti ini, tapi tetap ada sidebarnya

<img width="779" alt="image" src="https://user-images.githubusercontent.com/106010284/220309964-848a44e7-0836-422d-a770-a15421ee980a.png">

Saya coba buat kodenya :
<img width="439" alt="image" src="https://user-images.githubusercontent.com/106010284/220310452-8ff54f13-fabe-4fe6-8223-5bd018b79eed.png">

Tampilannya malah jadi seperti ini : 
<img width="897" alt="image" src="https://user-images.githubusercontent.com/106010284/220310557-45d0aa64-767d-40f3-81ec-700d12a5d04e.png">

Mohon bantuannya kak, terima kasih.

**Comment by finesaaa** [comment]
Dear Pak, @yohanespm77 
Thank you for the question; we've accepted your question.
However, we'll still try to help search the other references too. Please wait for this

**Comment by divakartika** [comment, marked as answer]
Dear @yohanespm77 ,

Sesuai diskusi pada [Rstudio Community ini](https://community.rstudio.com/t/how-do-i-add-tabs-to-the-dashboardheader/39674/2), membuat tab pada header tidak dapat dilakukan menggunakan `shinydashboard`. Meskipun demikian, hal tersebut dapat dicapai dengan menggunakan `navbarPage()` dari library `shiny`.

Contoh hasil: https://shiny.rstudio.com/gallery/navbar-example.html
Code: https://github.com/rstudio/shiny-examples/tree/main/052-navbar-example

Silakan dicoba dan dieksplor, semoga membantu :)

Best regards,
Team Algoritma

**Comment by yohanespm77** [comment]


**Reply by finesaaa** [reply]
Silahkan dapat bereksplorasi pada referensi berikut pak, seharusnya hampir sama dengan yang bapak inginkan 
https://stackoverflow.com/questions/46145608/how-to-combine-top-navigation-navbarpage-and-a-sidebar-menu-sidebarmenu-in-s
